src/main/resources/db/migration/changes/ZZ-LinkingTableChanges.json

```diff
@@ -0,0 +1,47 @@
+{
+    "databaseChangeLog": [
+        {
+            "changeSet": {
+                "id": "ZZ-1",
+                "author": "matt (generated)",
+                "changes": [
+                    {
+                        "createIndex": {
+                            "columns": [
+                                {
+                                    "column": {
+                                        "name": "CREATED_BY_ID"
+                                    }
+                                }
+                            ],
+                            "indexName": "FKKJPYGUUYD5SHXTABV9V5JPE6X_INDEX_2",
```

Comment:
Ideally an index name should be descriptive of what is being indexed, especially if we're manually maintaining it.

That said, is this the name of the existing index in the database? If so, I'd leave a comment here describing that fact.